apis/apis.json

```diff
@@ -901,6 +905,7 @@
       "listingDescription": "Firestore high-level library",
       "version": "2.3.1",
       "type": "other",
+      "metadataType": "VENEER_GRPC",
```

Comment:
If I remember correctly this was going to be replaced by GAPIC_COMBO.

Reply:
It's unclear - because for something like Firestore, it's *not* a GAPIC-like surface. I've asked for more clarity on this in the doc.

I'll merge this first and then update the ResourceManager PR to include the metadata type for that as well.

Reply:
No need to update ResourceManager v3 - we don't create a repo metadata file for that yet, as there's no release.